Add support for Stockport, UK

MuddyRock opened the pull request:
I may not have created the pull-request correctly - apologies if that's the case. I'm still a bit new to GitHub and Python.
There is probably a better way of iterating through the results than the one I went for. See above. Any feedback welcome.

MuddyRock commented:
This requires a Unique Property Reference Number (uprn) to determine the URL to use.
You can find your UPRN by going to 
https://www.stockport.gov.uk/find-your-collection-day
and entering your postcode (if you're testing this, and you live outside Stockport, try SK6 3AA), then picking your house number.
The UPRN is the final part of the URL that you are then directed to e.g.
for SK6 3AA, house number 25, you are directed to 
https://myaccount.stockport.gov.uk/bin-collections/show/10090543805
The UPRN is 10090543805

mampfes commented on custom_components/waste_collection_schedule/waste_collection_schedule/source/stockport_uk.py:
```diff
@@ -0,0 +1,74 @@
+            _LOGGER.error(
+                "uprn must be provided in config"
+            )
+        self._session = requests.Session()
```
Don't create session in __init__, but in fetch.

mampfes commented on custom_components/waste_collection_schedule/waste_collection_schedule/source/stockport_uk.py:
```diff
@@ -0,0 +1,74 @@
+                Collection(
+                    date=bin_date,
+                    t=bin_name,
+                    icon=ICONS[bin_name],
```
`icon=ICONS.get(bin_name)`
This variant doesn't crash if the bin name is missing in the dict, but fall back to default.

mampfes reviewed (changes requested):
- Please rename source to stockport_gov_uk
- Please add corresponding documentation as markdown file
- Please add links to new source in README.md and info.md
--> https://github.com/mampfes/hacs_waste_collection_schedule#how-to-add-new-sources

MuddyRock commented:
All requested changes made.
Specifically:
- Moved  create session to fetch
- Used correct method to check for icons
- Updated info.md
- Updated README.md
- renamed file to stockport_gov_uk.py
- Added doc file stockport_gov_uk.md in doc\source

I can't see how to re-request approval from you in GitHub for the changes.
Do I create a new pull request?

mampfes commented:
The diff for this PR shows major changes in README.md and info.md - maybe a problem with line endings. Check your editor and update README.md and info.md so that is shows only the newly added lines.

mampfes commented:
I did the requested changes in #622

mampfes commented:
Thanks for your contribution.
